airbyte-integrations/connectors/source-jdbc/src/main/resources/jdbc_models/jdbc_models.yaml

```diff
@@ -10,7 +10,7 @@ properties:
 definitions:
   JdbcState:
     type: object
-    additionalProperties: false
+    additionalProperties: true
```

Comment:
Why are we changing this file? I don' think we need since state is purely internal to the connector.